chapters/missing-data.qmd

```diff
@@ -0,0 +1,598 @@
+---
+knitr:
+  opts_chunk:
+    cache.path: "../_cache/missing-data/"
+---
+
+# Missing Data {#sec-missing-data}
+
+```{r}
+#| label: missing-data-setup
+#| include: false
+#| message: false
+#| warning: false
+
+source("../R/_common.R")
+
+# ------------------------------------------------------------------------------
+# Required packages
+
+library(tidymodels)
+library(rpart)
+library(partykit)
+
+# ------------------------------------------------------------------------------
+# Set options
+
+tidymodels_prefer()
+theme_set(theme_transparent())
+set_options()
+
+# ------------------------------------------------------------------------------
+# Load other data
+
+load("../RData/ames_original.RData")
+load("../RData/hotel_rates_all.RData")
+```
+
+```{r}
+#| label: missing-data-summaries
+#| cache: true
+#| message: false
+#| warning: false
+
+#Note:  Remove quality predictors. These are subjective and related to Sale Price
+ames_original <-
+  ames_original %>%
+  dplyr::select(!contains("Qual")) %>%
+  mutate(Neighborhood = factor(Neighborhood)) %>%
+  arrange(Neighborhood, Order)
+
+ames_predictors <-
+  ames_original %>%
+  dplyr::select(-Order, -PID, -SalePrice)
+
+ames_pct_missing <- 
```

Comment:
Some Ames predictors should be condensed (in terms of missingness). For example, basement finish type is really one (multiple-choice) variable. 

If you agree, I'll commit that later this week (or next week)

Reply:
I agree.